### DISCUSSION TITLE
How Advancing Earth Analytics Can Mitigate Catastrophic Events

### DISCUSSION THREAD

**Discussion by annieM459** [earthlab-education/Earth-Analytics-2023-01-Intro#88] [Awesome science]
Cool article and tool: https://www.heavy.ai/blog/how-advancing-earth-analytics-can-mitigate-catastrophic-events

**Comment by NobodyLivesHere** [comment]
This is cool.  Limitations in data processing has been one of the biggest setbacks for climate scientists in the past few decades.   It will be interesting to see how much predictive models improve for the effects of anthropogenic climate changes now that AI has come far enough to alleviate a lot of the human legwork and computing constraints.
